frontend/src/_includes/partials/audio-tags.html

```diff
@@ -0,0 +1,23 @@
+<div title="Unique file identifier">#{{ audioObj.fileId }}</div>
```

Comment:
Don't include `obj` in the parameter name. It's ugly and redundant.

Do it everywhere.

Reply:
Also, move the surrounding div here as well, it's a non-separable part of the tags partial.

Reply:
done